scripts/tools/InitializeQueue.php

```diff
@@ -66,9 +67,17 @@ class InitializeQueue extends InstallAction
 {
     use ServiceLocatorAwareTrait;
 
-    const BROKER_MEMORY = 'memory';
-    const BROKER_RDS = 'rds';
-    const BROKER_SQS = 'sqs';
+    public const BROKER_MEMORY = 'memory';
+    public const BROKER_RDS = 'rds';
+    public const BROKER_NEW_SQL = 'newsql';
+    public const BROKER_SQS = 'sqs';
+
+    public const AVAILABLE_BROKERS = [
+        self::BROKER_MEMORY,
+        self::BROKER_RDS,
+        self::BROKER_NEW_SQL,
+        self::BROKER_SQS,
+    ];
```

Comment:
```suggestion
    private const BROKER_MEMORY = 'memory';
    private const BROKER_RDS = 'rds';
    private const BROKER_NEW_SQL = 'newsql';
    private const BROKER_SQS = 'sqs';

    private const AVAILABLE_BROKERS = [
        self::BROKER_MEMORY,
        self::BROKER_RDS,
        self::BROKER_NEW_SQL,
        self::BROKER_SQS,
    ];
```

Those constants are only used here. We should follow the least privilege principle.

Reply:
done